data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/deserializers/HvdcRangeArrayDeserializer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_io_json.deserializers;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.data.crac_api.range_action.HvdcRangeActionAdder;
+import com.farao_community.farao.data.crac_api.range_action.HvdcRangeAdder;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+
+import java.io.IOException;
+
+import static com.farao_community.farao.data.crac_io_json.JsonSerializationConstants.*;
+
+/**
+ *  @author Godelaine de Montmorillon {@literal <godelaine.demontmorillon at rte-france.com>}
+ */
+public final class HvdcRangeArrayDeserializer {
+    private HvdcRangeArrayDeserializer() {
+    }
+
+    public static void deserialize(JsonParser jsonParser, HvdcRangeActionAdder ownerAdder) throws IOException {
+        while (jsonParser.nextToken() != JsonToken.END_ARRAY) {
+            HvdcRangeAdder adder = ownerAdder.newHvdcRange();
+            while (!jsonParser.nextToken().isStructEnd()) {
+                switch (jsonParser.getCurrentName()) {
+                    case MIN:
+                        adder.withMin(jsonParser.nextIntValue(Integer.MIN_VALUE));
```

Comment:
I would not put a default value here. If we cannot parse 'nextIntValue', it means that there is a problem in the JSON file. Same comment 4 lines below